lib/mpl_toolkits/axes_grid1/anchored_artists.py

```diff
@@ -1,43 +1,156 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+from matplotlib import docstring
 from matplotlib.externals import six
-
+from matplotlib.offsetbox import (AnchoredOffsetbox, AuxTransformBox,
+                                  DrawingArea, TextArea, VPacker)
 from matplotlib.patches import Rectangle, Ellipse
 
-import numpy as np
-
-from matplotlib.offsetbox import AnchoredOffsetbox, AuxTransformBox, VPacker,\
```

Comment:
The code in `if __name__ == "__main__":` is the only place where `AnchoredText` and `AnnotationBbox` are used, so I figured that as the reason for their being imported. Looking further into it, both are imported in `lib/mpl_toolkits/axes_grid/anchored_artists.py`, so maybe they are meant to be part of the axes_grid{,1} APIs.

Strangely, `AnnotationBbox` starts in `matplotlib.offsetbox` in 80fb71a76d9fc03795d182bf1c58e9ca70b1501e despite being added in axes_grid (`AnchoredText` is also moved in that commit without explaination). The demo for it doesn't use axes_grid at all.

I'll re-import those classes and add the imports to `__all__`, since I think a cleanup of the API is outside the scope of this PR.